README.md

```diff
@@ -21,6 +21,19 @@ If you want to install [`clickhouse-cityhash`](https://pypi.org/project/clickhou
 > pip install asynch[compression]
 ```
 
+## Release v0.3.0 announcement
+
+The version 0.2.5 should have been named v0.3.0 due to compatibility-breaking changes.
+Since the release v0.2.5 is already yanked (of sorts), a smooth upgrade seems inexpedient.
+
+Before upgrading to the v0.3.0, please pay attention to the incompatible changes:
+
+1. the `connect` function from the `async/connection.py` module is now asynchronous context manager.
+2. the same kind of changes touches the `create_pool` function from the `async/pool.py` module.
+3. all deprecated methods from `Connection`, `Cursor` and `Pool` classes are removed.
+
+For more details, please refer to the project [CHANGELOG.md](./CHANGELOG.md) file.
```

Comment:
@long2ice , are you OK with this? Any suggestions on improving this announcement?